ui/jetbrains/src/main/java/com/neueda/jetbrains/plugin/graphdb/jetbrains/actions/ui/console/ToggleFileSpecificParametersUsedAction.java

```diff
@@ -0,0 +1,40 @@
+package com.neueda.jetbrains.plugin.graphdb.jetbrains.actions.ui.console;
+
+import com.intellij.icons.AllIcons;
+import com.intellij.openapi.actionSystem.AnAction;
+import com.intellij.openapi.actionSystem.AnActionEvent;
+import com.intellij.openapi.actionSystem.Presentation;
+import com.neueda.jetbrains.plugin.graphdb.jetbrains.component.settings.SettingsComponent;
+import org.jetbrains.annotations.NotNull;
+
+public class ToggleFileSpecificParametersUsedAction extends AnAction {
+
+    public ToggleFileSpecificParametersUsedAction() {
+
+    }
+
+    @Override
+    public void actionPerformed(@NotNull AnActionEvent e) {
+        SettingsComponent settings = SettingsComponent.getInstance();
+        boolean setToUseFileSpecificParams = !settings.areFileSpecificParamsUsed();
```

Comment:
Bad name for boolean `set`. Use name of method on the right side?

Reply:
fixed in [d787900e](https://github.com/ex3m1024/jetbrains-plugin-graph-database-support/commit/d787900e73bc18d48e9b0ac186e839f178cb809b)